### PR TITLE
fix(#9429): pass 500 reports to forms contact summary

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

#9429

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:9429-all-reports-in-contact-summary/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:9429-all-reports-in-contact-summary/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:9429-all-reports-in-contact-summary/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
Hi @lorerod and @latin-panda 

This is another high priority urgent issue holding off 4.11 for digital payments :( 
I'd love feedback from both of you! Thanks a lot :)

**Comment by kennsippell** [comment]
Tested with digital payment data. Works great. Thanks!

**Comment by dianabarsan** [comment]
@kennsippell 
I hope you tested with latest code and by checking out the branch yourself. Otherwise, please wait until last commit pass CI so you can install latest code. Thanks

File: tests/e2e/default/enketo/contact-summary-reports.wdio-spec.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    await utils.updateSettings({ contact_summary: contactSummary }, true);
+
+    await utils.saveDocs([...places.values(), patient, ...reports]);
+
+    await cookieLogin();
+  });
+
+  it('should load all reports', async () => {
+    await commonPage.goToPeople(patient._id);
+    await browser.pause(5437854353);
```
Comment (latin-panda) [inline tests/e2e/default/enketo/contact-summary-reports.wdio-spec.js:42]: This is a funny number 😅 but I understand it takes time to load so many. 
Can you please add a comment? 
```suggestion
    await browser.pause(5437854353); // Waiting to calculate and load 100 reports
```

**Review by latin-panda — APPROVED** [review]
This is working as expected. I tested with 550 reports, the contact summary only gets 500 max. Then with 400 reports, the contact summary gets the 400. 

Thanks Diana✨ ready to merge 🚀

**Comment by lorerod** [comment]
Sorry, I was late for this. Thanks, @dianabarsan, @kennsippell @latin-panda.

**Pull request merged**